uportal-war/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp

```diff
@@ -139,8 +139,8 @@
         </a>
         <c:forEach var="image" items="${thumbnailImages}" varStatus="status">
             <a href="#">
-                <img src="${image}" />
-                <span class="caption">Background ${status.index + 1}</span>
+                <img src="${image}" alt=${imageCaptions[status.index]}/>
+                <span class="caption">${imageCaptions[status.index]}</span>
```

Comment:
Bigger picture: background images are decoration, right? What's the use case for screen-reader users to select among decorative background images? Would screen reader users be better served by this feature (both the selection of decorative background images, and the presentation of decorative background images) responsively degrading out of the product when used on a screen reader?

That is, screen reader users may not be best served by being given more information to inform selecting a background image. They might be best served by not being bothered about background images? Is this an admin-facing user, a use case of a screen-reader-using tenant administrator selecting a background image for use by non-screen-reader-using tenants?